packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -57,13 +57,18 @@ const PROVIDER_TYPE_OPTIONS = PROVIDER_TYPES.map((type) => ({
 
 const oVirtLabelPrefix = process.env.BRAND_TYPE === 'RedHat' ? 'RHV Manager' : 'oVirt Engine';
 const vmwareLabelPrefix = 'vCenter';
-const getLabelName = (type: 'hostname' | 'username' | 'pwd', prefix?: string) => {
+const openStackLabelPrefix = 'OpenStack Identity (Keystone)';
+const getLabelName = (type: 'hostname' | 'URL' | 'username' | 'pwd', prefix?: string) => {
   let label = '';
   switch (type) {
     case 'hostname': {
       label = prefix ? `${prefix} host name or IP address` : 'Host name or IP address';
       break;
     }
+    case 'URL': {
+      label = prefix ? `${prefix} URL` : 'URL';
+      break;
+    }
```

Comment:
you don't need it

Reply:
This is needed for the `brandPrefix` to work (please check my previous comment https://github.com/kubev2v/forklift-console-plugin/pull/206#discussion_r1108742425)